pkg/core/store_option.go

```diff
@@ -259,6 +259,15 @@ func ResetStoreLimit(limitType storelimit.Type, ratePerSec ...float64) StoreCrea
 	}
 }
 
+// SetStoreLimit set the store for a store, it may switch the store limit mode.
+func SetStoreLimit(limit storelimit.StoreLimit) StoreCreateOption {
```

Comment:
only test use it, am I right?

Reply:
yes in past， It will be used in switch store limit v2.